Common/package.json

```diff
@@ -18,43 +18,32 @@
         "rebuild": "npm run clean && npm run build",
         "watch:proto": "nodemon -V -d 3 --watch protobuf/signalling_messages.proto --exec \"npm run build:proto\"",
         "watch": "concurrently -k \"npm run watch:proto\" \"tsc --watch --preserveWatchOutput --project tsconfig.cjs.json\"",
-        "lint": "eslint src --ext .js,.jsx,.ts,.tsx",
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "wtf": "concurrently -k \"npm list\""
+        "lint": "eslint src",
+        "test": "echo \"Error: no test specified\" && exit 1"
     },
     "devDependencies": {
-        "@types/jest": "^29.5.14",
-        "@types/webxr": "^0.5.1",
-        "@typescript-eslint/eslint-plugin": "^6.21.0",
+        "@protobuf-ts/plugin": "^2.9.4",
```

Comment:
Protobuf seems to be used in source itself in `Common\src\Messages\signalling_messages.ts`. I am thinking it is a dep?